static/src/stylesheets/inline/story-package-garnett.scss

```diff
@@ -28,7 +37,7 @@
 }
 
 .fc-container--story-package {
-    background-color: #6b6e71; // This colour does not exist (in the palette)
+    background-color: $story-package-container-colour; // This colour does not exist (in the palette)
```

Comment:
Super nit-picky, but what do you think about moving this comment to the variable definition?